third_party/amd/lib/TritonAMDGPUTransforms/ConvertToBufferOps.cpp

```diff
@@ -491,10 +739,12 @@ class TritonAMDGPUConvertToBufferOpsPass
       if (op->getOperand(0).getDefiningOp<arith::CmpIOp>())
         assumptions.insert(op->getOperand(0));
     });
-    LDBG("Number of assumptions found: " << assumptions.size());
-    for (Value assume : assumptions) {
-      LDBG("Assumption:" << assume);
-    }
+
+    std::shared_ptr<DataFlowSolver> solver = createDataFlowSolver();
+    solver->load<TritonIntegerRangeAnalysis>();
```

Comment:
I think you'll still want to pull in DeadCodeAnalysis, unless the override on visiting region successors makes that not needed anymore?

Reply:
That's already pulled in inside of `createDataFlowSolver()`.